lib/v2/screens/app/interactor/viewmodels/app_bloc.dart

```diff
@@ -29,6 +33,15 @@ class AppBloc extends Bloc<AppEvent, AppState> {
     if (event is ShouldShowNotificationBadge) {
       yield state.copyWith(hasNotification: event.value);
     }
+    if (event is HandleInitialDeepLink) {
```

Comment:
may be leave some comments explaning the difference between initial deep link and incoming deep link.

Reply:
added